src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,47 @@
+package mate.controller;
```

Comment:
Let's create a separate packages `driver`, `car`, `manufacturer` inside `controller` package and divide controllers between these packages